docker/compose.yml

```diff
@@ -33,7 +33,6 @@ services:
       - SMTP_MAIL_USER=username
       - SMTP_MAIL_PASSWORD=password
       - MAIL_FROM=admin@example.com
-      - NEXT_PUBLIC_ALLOW_SIGNUP=true
```

Comment:
So, the new env var `NEXT_PUBLIC_DISABLE_SIGNUP` is not needed here anymore?

Reply:
not by default no

Reply:
Ok, thanks!